build/tools/changelog.sh

```diff
@@ -17,7 +17,11 @@
 
 Changelog=Changelog.txt
 
-DEVICE=$(echo $TARGET_PRODUCT | sed -e 's/lineage_//g')
+if [ -z $1 ]; then
```

Comment:
based on https://github.com/crdroidandroid/android_build/pull/33/commits/42577fb2d642f9cd37c0a2639d21c66441623bbe
I think checking if set is redundant